### DISCUSSION TITLE
Emphasize dimension in AT6

### DISCUSSION THREAD

**Discussion by StevenClontz** [TeamBasedInquiryLearning/linear-algebra#298] [General]
I think in [Fact 3.6.3](https://teambasedinquirylearning.github.io/linear-algebra/2024/AT6.html#fact-18) 
 we should emphasize that $V$ is isomorphic to $\mathbb R^n$ if it has dimension $n$. Then in activities [Activity 3.6.4](https://teambasedinquirylearning.github.io/linear-algebra/2024/AT6.html#activity-115) and [Activity 3.6.5](https://teambasedinquirylearning.github.io/linear-algebra/2024/AT6.html#activity-116) we can scaffold by asking the dimension of each space before asking which set they are isomorphic to.

**Comment by siwelwerd** [comment]
I think that's fine. As I read through 3.6 though, I wonder if we need to say more about why we need a bijection to have an isomorphism.  Would be easier if we had the language of invertibility, but not sure it's worth rearranging.  

To elaborate, I see how [Activity 3.6.2](https://teambasedinquirylearning.github.io/linear-algebra/2024/AT6.html#activity-114) gives the idea of a proof of how to construct a bijection between vector spaces with the same dimension.  But a natural question from a student might be why can't I just make an injection, or a surjection?

**Reply by fragandi** [reply]
It is a good observation and in fact something that I think it is missing from the current version of the book, that if you have a map between vector spaces of the space dimension, injective (or surjective) is enough to give bijective. In a sense this is why we want a concept of dimension: to make ideas from discrete/finite sets still work in the "infinite world". 

An issue that I am having with some of the materials from my institution is that this observation is made too early and then students have difficulties distinguishing between injective and surjective because it all gets mushed together for nxn matrices. I am making this remark just to say that I think this is an idea that should appear, but not too early, so AT6 might be a good spot for it.

**Reply by siwelwerd** [reply]
Right, I think I always wanted to avoid making that observation because of that exact confusion.  I agree this might be the place to add it though.

**Comment by StevenClontz** [comment]
Thanks - I've created an [issue](https://github.com/TeamBasedInquiryLearning/linear-algebra/issues/300) to implement this at some point.
